app/src/main/res/layout/fragment_profile.xml

```diff
@@ -19,25 +19,39 @@
         android:layout_height="match_parent"
         android:orientation="vertical">
 
-        <TextView
-            android:id="@+id/name"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="@dimen/padding_extra_large"
-            android:text="@string/user_name"
-            android:textAlignment="center"
-            android:textColor="@color/black"
-            android:textSize="@dimen/text_size_expanded_title" />
-
         <ImageView
             android:id="@+id/avatar"
             android:layout_width="@dimen/avatar_medium"
             android:layout_height="@dimen/avatar_medium"
             android:layout_gravity="center_horizontal"
-            android:layout_marginBottom="@dimen/layout_margin_small"
+            android:layout_marginTop="@dimen/padding_extra_large"
             android:scaleType="centerCrop"
             app:srcCompat="@drawable/ic_account_circle_grey_24dp" />
 
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <TextView
+                android:id="@+id/name"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/user_name"
+                android:layout_centerInParent="true"
+                android:textColor="@color/black"
+                android:textSize="@dimen/text_size_expanded_title" />
+
+            <ImageButton
+                android:id="@+id/editProfileButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:background="@null"
+                android:layout_toRightOf="@+id/name"
+                android:layout_marginLeft="@dimen/layout_margin_large"
+                android:layout_centerInParent="true"
+                app:srcCompat="@drawable/ic_create_black_24dp" />
+        </RelativeLayout>
```

Comment:
Replace with LinearLayout

Reply:
> Replace with LinearLayout

Is it a good idea to use LinearLayout when you have a TextView and an ImageButton at the same row? I mean the layout in the row will be spoiled. Sorry if that's a silly question.

Reply:
It won't be spoiled. In fact, this layout is so simple that it doesn't even need to be a relative layout

Reply:
> It won't be spoiled. In fact, this layout is so simple that it doesn't even need to be a relative layout

Thank you for the help! I made all the necessary corrections